slm_porosity/heat_transfer.f90

```diff
@@ -112,6 +113,9 @@ END FUNCTION func_with_derivative
   INTEGER  :: thickness_points
   INTEGER  :: j_mx_porosity
   REAL(pr) :: cfl_fusion_factor
+  LOGICAL  :: symmetry_flag
```

Comment:
1. `flag` is tautology. We already use `LOGICAL` to define this variable as boolean flag.
2. `symmetry` is a quality of the problem, but it is not about, how we deal with this quality.